.github/workflows/release.yml

```
@@ -1,143 +1,321 @@
-                      shutil.copy2(src, artifact_dir / dest_name)
-          PY
-          ls release-assets
-          if compgen -G "release-assets/*.bin" >/dev/null; then
-            (
-              cd release-assets
-              sha256sum *.bin > checksums.txt
-            )
+      - name: Generate Release Notes
+        id: release_notes
+        shell: bash
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2129:style:5:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)